crates/vhost-user-backend/tests/vhost-user-server.rs

```diff
@@ -154,7 +154,7 @@ fn vhost_user_client(path: &Path, barrier: Arc<Barrier>) {
     )
     .unwrap();
     // SAFETY: Safe because we panic before if memfd is not valid.
-    let file = unsafe { File::from_raw_fd(memfd) };
+    let file = unsafe { File::from_raw_fd(memfd.as_raw_fd()) };
```

Comment:
I don't think that's the proper way to do it. Given `File` implements `From<OwnedFd>`, you can simply do the following:
```suggestion
    let file = File::from(memfd);
```

Reply:
Ooops, right!